specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-08-02-preview/CommonDefinitions.json

```diff
@@ -0,0 +1,72 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-08-02-preview",
+    "title": "AppPlatformManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {},
+  "definitions": {
+    "SpringAppsTemplate": {
+      "description": "Azure Spring Apps App Template.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../app/resource-manager/Microsoft.App/preview/2023-04-01-preview/CommonDefinitions.json#/definitions/Template"
```

Comment:
why you ref to another rp's schema?

Reply:
@visingla-ms, I saw you added the label of 'ARMSignoff', but I doubt whether we should allow the cross reference from one RP to another RP.  The only exception is the Common Types defined by ARM team.

In this case, the change in Microsoft.App may break the API/SDK of Microsoft.AppPlatform, but we are unable to detect that in our tooling.

Reply:
This is a really good point. I'd highly recommend just referencing common-types. And if what you need isn't actually there, we can work on adding it.